libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/FormatUtils.java

```diff
@@ -32,4 +32,27 @@ public static final String formatInt(int value) {
     public static final String formatDecimal(int value) {
         return DECIMAL_INSTANCE.get().format(value).toString();
     }
+
+    /*
+     * returns the passed long formatted has an human readable filesize. Ex: 10 GB
+     */
+    public static final String formatFileSize(long size) {
+        if (size <= 0) {
+            return "0";
+        }
+        final String[] units = new String[] {"B", "kB", "MB", "GB", "TB" };
+        int digitGroups = (int) (Math.log10(size) / Math.log10(1024));
+
+        return new DecimalFormat("#,##0.#").format(size / Math.pow(1024, digitGroups)) + " " + units[digitGroups];
```

Comment:
concatenating strings might not be the best idea i18n-wise (for a good explanation, see https://github.com/wordpress-mobile/WordPress-Android/pull/6713#issuecomment-347906050)

Reply:
It's true, for example in Japanese there is no space between size and name `30キロバイト` (30 kilobytes).

Could we do something like "%d B" (context: `file_size_in_bytes`),  "%d kB" (context: `file_size_in_kilobytes`), etc? And then replace the `%d` with `#,##0.#` to format it?

Also, I am pretty sure you should rather use `DECIMAL_INSTANCE` instead of `new DecimalFormat`. It might be outside of the scope of this PR but we should actually initialize `DecimalFormat` with the Locale so that things like decimal separators are appropriately chosen per language (for example German uses a comma instead of a period for decimals).